README.md

```diff
@@ -55,6 +55,7 @@ The following Query Parameters are supported:
   - You can specify at most 100 wanted collections/prefixes.
 - `wantedDids` - An array of Repo DIDs to filter which records you receive on your stream (Default empty = all repos)
   - You can specify at most 10,000 wanted DIDs.
+- `maxSize` - The maximum size of a payload that this client would like to receive. Zero means no limit, negative values are treated as zero. (Default "0" or empty = no maximum size)
```

Comment:
not immediately clear how you specify the size i.e. bytes? megabytes? etc.

Reply:
like further down i see this is bytes, but would be good to make it clearer here